docs/business.yaml

```diff
@@ -1213,6 +1273,51 @@ paths:
                           actions:
                             - addressChanged
                             - nameChanged
+              change-of-officers-request:
+                summary: Change Of Officers Request
+                value:
+                  filing:
+                    header:
+                      name: changeOfOfficers
```

Comment:
can we add `changeOfOfficers` to the [name property](https://github.com/bcgov/lear/blob/152affb74dbf23641b77375cad719220cfb137cc/docs/business.yaml#L5487-L5524) of the filing header schema?